charts/dapr/Chart.yaml

```diff
@@ -1,24 +1,24 @@
 apiVersion: v1
-appVersion: "1.0.0"
+appVersion: "1.1.0-rc.1"
 description: A Helm chart for Dapr on Kubernetes
 name: dapr
-version: 1.0.0
+version: 1.1.0-rc.1
 dependencies:
   - name: dapr_rbac
-    version: "0.6.0"
+    version: "1.1.0-rc.1"
```

Comment:
0.7.0? same for others ... one plus the current?

Reply:
@mukundansundar Is there any reason why we wouldn't version the charts congruently with the release?

Reply:
I like the idea to keep helm chart versions in sync with the tag. @yaron2 ?

Reply:
I think unfortunately the chart version (0.7.0) has surpassed the tag version (0.6.0) so we are out of sync on this one. Thoughts?